src/app/wizard/wizard.component.html

```diff
@@ -39,20 +39,20 @@ <h3 class="blank-slate-pf-main-action">{{config?.loadingTitle}}</h3>
 </div>
 <div class="modal-footer wizard-pf-footer pfng-wizard-position-override"
      [ngClass]="{'pfng-footer-inline': config?.embedInPage}">
-  <button class="btn btn-default btn-cancel wizard-pf-cancel" type="button"
-          [ngClass]="{'wizard-pf-cancel-no-back': config?.hidePreviousButton}"
+  <button class="btn btn-default wizard-btn" type="button"
           [disabled]="config?.done"
           (click)="cancel()"
           *ngIf="!config?.embedInPage">{{config?.cancelTitle}}</button>
-  <button class="btn btn-default" type="button"
+  <button class="btn btn-default wizard-btn" type="button"
```

Comment:
Perhaps this should be renamed pfng-wizard-previous-btn, since it's specific to the previous button? 

The 'pfng' prefix simply indicates that it's a style in patternfly-ng and not patternfly.